platforms/hyperledger-indy/configuration/roles/check/validation/tasks/check_count.yaml

```diff
@@ -0,0 +1,36 @@
+- name: "Counting Genesis Steward"
+  set_fact:
+    steward_count={{ steward_count|default(0)|int + 1 }}
+  loop: "{{ stewards }}"
+
+- name: Set trustee count to zero
+  set_fact:
+    trustee_count=0
+  
+- name: "Counting trustees per Org"
+  set_fact:
+    trustee_count={{ trustee_count|default(0)|int + 1 }}
+    total_trustees={{ total_trustees||default(0)int + 1 }}
+  loop: "{{ trustees }}"
+
+- name: Print error if trustee count limit fails
+  debug: msg="The trustee count is {{ trustee_count }}. There should be max 1 trustee per organization."
```

Comment:
Will this fail the playbook? use failed_when, check https://docs.ansible.com/ansible/latest/user_guide/playbooks_error_handling.html

Reply:
Yes, if the condition is unsatisfied , it will fail and the playbook execution will be stopped. It will give the error msg and will not proceed any furthur.